modules/imgproc/src/resize.cpp

```diff
@@ -50,12 +50,1326 @@
 #include "precomp.hpp"
 #include "opencl_kernels_imgproc.hpp"
 #include "hal_replacement.hpp"
+#include "opencv2/core/hal/intrin.hpp"
 
 #include "opencv2/core/openvx/ovx_defs.hpp"
 #include "resize.hpp"
 
+#include "opencv2/core/softfloat.hpp"
+
 using namespace cv;
 
+#ifndef CV_ALWAYS_INLINE
```

Comment:
Check for existed "CV_INLINE" macro.

Reply:
CV_INLINE doesn't force inlining and could add `static` modificator,